File: python/7 kyu/323.py
# https://www.codewars.com/kata/5a8059b1fd577709860000f6/train/python

# Passed

def alphabetic(s):
    s = s.lower()
    result = True
    for idx in range(len(s) - 1):
        if ord(s[idx]) > ord(s[idx + 1]):
            result = False
            break
        
    return result

output = alphabetic('door')
print(output)

File: nodejs/7 kyu/323.js
// https://www.codewars.com/kata/555b73a81a6285b6ce000047/train/javascript

// Passed

function add(...args) {
  let multiplier = 1;
  let result = 0;
  for (const num of args) {
    result += multiplier * num;
    multiplier++;
  }

  return result;
}

const output = add(100, 200, 300);
console.log(output);